opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -115,10 +115,7 @@ message ProfilesDictionary {
   repeated string string_table = 5;
 
   // A common table for attributes referenced by various messages.
-  repeated opentelemetry.proto.common.v1.KeyValue attribute_table = 6;
-
-  // Represents a mapping between Attribute Keys and Units.
-  repeated AttributeUnit attribute_units = 7;
+  repeated KeyValueAndUnit attribute_table = 6;
```

Comment:
I was under the impression the Profiling SIG settled on the common `KeyValue` to be consistent with other signals. Has this decision been changed?